src/lib/hooks/swap/useSwapInfo.tsx

```diff
@@ -55,10 +55,7 @@ function useComputeSwapInfo(): SwapInfo {
     [inputCurrency, isExactIn, outputCurrency, amount]
   )
 
-  /**
-   * @TODO (ianlapham): eventually need a strategy for routing API here
```

Comment:
Is this comment resolved?

Reply:
Kinda depends, id say its more of a product decision now if we want to use the API, but now that we have SOR we aren't missing anything significant - ill leave a note that this is where routing API logic would go